internal/operands/common-templates/reconcile.go

```diff
@@ -35,10 +35,11 @@ func init() {
 	utilruntime.Must(templatev1.Install(common.Scheme))
 }
 
-func WatchClusterTypes() []client.Object {
-	return []client.Object{
-		&templatev1.Template{},
-	}
+func WatchClusterTypes() []operands.WatchType {
+	return []operands.WatchType{{
+		Object:          &templatev1.Template{},
+		WatchFullObject: true,
```

Comment:
`WatchFullObject = true` is the default?

Reply:
No, `false` is default.